docs/en/configuration.md

```diff
@@ -7,17 +7,30 @@ will interfere with locale bootstrapping in certain situations (such as `Securit
 
 ## Locale configuration
 
-You can create locales via the `/admin/locales` CMS section. Once you add at least two
-locales to your site, you can begin localising your content.
+You can create locales via the `/admin/locales` CMS section. 
 
 Each locale has these fields in the CMS editor:
  - `Locale`: Dropdown which lets you select a locale code from the global locale source
  - `Title`: Name to use for this locale in the locale switcher
  - `URL Segment`: Defaults to the locale (e.g. `en_NZ`) but can be customised. Must be unique.
- - `Default`: Check this to set as the global default. Note: If using domains, you can
-   additionally assign per-domain defaults as well.
+ 
+Check the box titled `This is the global default locale` to set this locale as the global default. 
+ 
+**Note:** If using domains, you can additionally assign per-domain defaults as well.
+
  - `Domain`: Dropdown to assign this locale to a domain.
- - `Fallbacks`: Allows you to specify one or more fallback locales for this locale.
+ 
+Navigate to the `Fallbacks` tab, which allows you to specify one or more fallback locales for this locale.
+
+Once you add at least two locales to your site, you can begin localising your content. 
+
+***Important:** Pages in locales that fall back must be added and published in each locale you want them to be visible 
```

Comment:
```diff
- ***Important:**
+ **Important:**
```

Reply:
As this point is rather important see #350, I intended to display this block in italics and bold. I am happy though to change it to something better suited - any ideas - to draw the necessary attention to it.

Reply:
Ah, I see - in that case you need to close it correctly (notice that the bold carries through). Maybe use an underscore for italic instead